manual/src/ornate/cookbook.md

```diff
@@ -325,3 +325,32 @@ You can use it e.g. with `[sbt-native-packager](https://github.com/sbt/sbt-nativ
 ~~~ scala
 mappings in (Compile, packageBin) ++= (webpack in (Compile, fullOptJS)).value.map { f => f.data -> f.data.getName() },
 ~~~
+
+## How to package for [Electron](https://electronjs.org)
+
+Packaging for Electron is actually not that hard either (but some more manual steps are needed to fully
+get this to work - e.g., writing Facades for Electron's built-in types.)
+
+### Add an extra Webpack configuration to your project
+
+A suggested location is `src/main/resources`. Create a file named `electron.webpack.config.js` there and add these lines:
+
+~~~ javascript
+module.exports = require('./scalajs.webpack.config');
+module.exports.target = 'electron';
+~~~
```

Comment:
Do you think you could add an electron compatible project in the tests? This way we could:

- test that the code you wrote here still works as scalajs-bundler and webpack evolve,
- insert into the documentation the code that is actually used in the test (like we do for the global namespace thing).